nuspec/Cake.Portable.nuspec

```diff
@@ -0,0 +1,31 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!-- Do not remove this test for UTF-8: if “Ω” doesn’t appear as greek uppercase omega letter enclosed in quotation marks, you should use an editor that supports UTF-8, not this one. -->
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>cake.portable</id>
```

Comment:
This is really in line with the guidance that we provide on the Chocolatey Wiki:

https://github.com/chocolatey/chocolatey/wiki/CreatePackages#naming-your-package

I used `choco new` to create the package which applies this guidance as well.  Although we can make it different, the result on chocolatey.org on the package page will be the lowercase version.  For example, the casing of this package, is GitVersion.Portable (I think it made it through before the guidance was finalised): https://chocolatey.org/packages/GitVersion.Portable is `GitVersion.Portable`, but is displays at `gitversion.portable` when shown in the installation step.

I can certainly make the ID capital, but depending on the moderation, it might be pushed back.